stock_operating_unit/view/stock.xml

```diff
@@ -156,24 +156,6 @@
         </field>
     </record>
 
-    <record id="view_move_picking_form" model="ir.ui.view">
```

Comment:
Thanks for migrating this.

Is the operating unit in stock move not visible anymore?

Reply:
The operating units are still visible in the views of `stock.move`. I removed this part of code because `stock.view_move_picking_form` doesn't exist anymore.

Reply:
All rgiht. Thanks for answering. I will wait for #229 to review this.